Gopkg.toml

```diff
@@ -0,0 +1,50 @@
+# Gopkg.toml example
```

Comment:
Generated and could be deleted.

Reply:
Yes could be deleted. Its quite good comment. Link to its documentation and some examples